apps/srp.c

```diff
@@ -553,6 +553,8 @@ int srp_main(int argc, char **argv)
             user = *(argv++);
         else {
             user = NULL;
+            if (mode == OPT_LIST)
+                break;
```

Comment:
The -list, -add, -modify and -delete options cannot be combined and as far as I got it the code checks this and errors out.